backend/spec/controllers/spree/admin/base_controller_spec.rb

```diff
@@ -22,5 +19,18 @@ def index
       get :index
       expect(response).to redirect_to '/unauthorized'
     end
+
+    context "when an unauthorized redirect handler is provided" do
+      before do
+        Spree.deprecator.silence do
+          Spree::Admin::BaseController.unauthorized_redirect = -> { redirect_to '/custom_unauthorized' }
```

Comment:
I think this introduced a test order dependent failure based on the spec failures on CI 👀

Reply:
Thank you @forkata , good find! This should be fixed now.